Source/WTF/wtf/text/AtomString.cpp

```diff
@@ -108,15 +108,15 @@ AtomString AtomString::number(unsigned long long number)
 AtomString AtomString::number(float number)
 {
     NumberToStringBuffer buffer;
-    size_t length = numberToStringAndSize(number, buffer);
-    return AtomString { std::span { bitwise_cast<const LChar*>(buffer.data()), length } };
+    auto span = numberToStringAndSize(number, buffer);
+    return AtomString { spanReinterpretCast<const LChar>(span) };
```

Comment:
Not sure how to use `byteCast<LChar>` here without re-introducing a `-Wunsafe-buffer-usage` warning.  (That's the reason `spanReinterpretCast<const LChar>()` is being used.)

Reply:
Does not make logical sense to me, because `byteCast` should,be identical to `spanReinterpretCasr` in this respect. If it's not, I'd want to figure out why and fix it.